src/components/Book/Book.tsx

```diff
@@ -1,15 +1,22 @@
-import { useState } from "react";
 import "./Book.scss";
 import { BookType } from "../BookList/BookList";
 
-export const Book = ({id, title, isbn, authors}: BookType) => {
+export const Book = ({ id, title, isbn, authors }: BookType) => {
   return (
     <div className="book">
-      <img className="image" src="./book-cover-placeholder.png" alt={title}></img>
+      <img
+        className="image"
+        src="./book-cover-placeholder.png"
+        alt={title}
+      ></img>
       <div className="title">{title}</div>
       <ul className="authors">
-      {authors &&
-        authors.map((author) => <li className="author">{author}</li>)}
+        {authors &&
+          authors.map((author, idx) => (
+            <li key={idx} className="author">
```

Comment:
every single author have id, we shouldnt use index

Reply:
we have an id for book, not author, we don't want duplicates in keys of an ul